tests/integration/targets/acme_letsencrypt/http-challenge-local.yml

```diff
@@ -60,3 +60,8 @@
         that:
           - "'DNS:example.de' in result.subject_alt_name"
           - "'Pebble Intermediate CA' in result.issuer.commonName"
+
+    - name: Remove account identifier, to avoid cache problems in CI
```

Comment:
I think you also observed flakiness in the tests. And I may have found the main issue here. The Pebble Server seems to cahce validations for specific accounts. If the test runs fast enough, the second mechanism will always run into errors. By deleting the acount key we skip this caching and I have not been able to reproduce flakyness in the tests.

Reply:
Good catch.